.env.staging

```diff
@@ -1,6 +1,6 @@
 NODE_ENV=production
-VUE_APP_SERVER_ROOT="http://167.99.228.44:3000"
-VUE_APP_SOCKET_ADDRESS="http://167.99.228.44:3001"
+VUE_APP_SERVER_ROOT="http://167.172.145.56:3000"
```

Comment:
this was just supposed to be a temporary change. i'm guessing we're going to kill mobile-staging at some point, so this should be switched back to staging ip. if you want to keep mobile-staging server, would need another env file